.gitignore

```diff
@@ -0,0 +1 @@
+node_module
```

Comment:
You are ignoring node_module, but should ignore "node_modules", the last letter is missing.

- [ ] Type `nome_modules` instead `node_module`.